packages/devextreme/js/__internal/ui/chat/chat_header.ts

```diff
@@ -0,0 +1,65 @@
+import type { Properties } from '@js/core/dom_component';
+import DOMComponent from '@js/core/dom_component';
+import type { dxElementWrapper } from '@js/core/renderer';
+import $ from '@js/core/renderer';
+
+const CHAT_HEADER_CLASS = 'dx-chat-header';
+const CHAT_HEADER_TEXT_CLASS = 'dx-chat-header-text';
+
+export interface ChatHeaderProperties extends Properties {
+  title?: string;
```

Comment:
Because this is not public interface or type, may be we might declare 'title' option the following way:
```
title: string;
```

Reply:
[61fdb892ea26ec5fe331835862bfb84dc42b9632]